pkgs/tools/security/enpass/default.nix

```diff
@@ -0,0 +1,86 @@
+{stdenv, fetchurl, dpkg, patchelf, openssl, xorg
+, glib, mesa, libpulseaudio, zlib, dbus, fontconfig, freetype
+, gtk2, pango, atk, cairo, gdk_pixbuf, jasper, xkeyboardconfig
+, makeWrapper , makeDesktopItem, python}:
+let
+  data = import ./data.nix;
+  # used of both wrappers and libpath
+  uselibs = with xorg; [
+    openssl
+    mesa
+    fontconfig
+    freetype
+    libpulseaudio
+    zlib
+    dbus
+    libX11
+    libXi
+    libSM
+    libICE
+    libXext
+    libXrender
+    libXScrnSaver
+    glib
+    gtk2
+    pango
+    cairo
+    atk
+    gdk_pixbuf
+    jasper
+  ];
+    libPath = stdenv.lib.makeLibraryPath ([
```

Comment:
Check your indentation - you're slipping into four spaces here where 2 spaces is common